### PR TITLE
[TASK] PL1-3735 Upgrade clix to use urfave/cli/v2

### PR DESCRIPTION
**Pull request by Skandalik**

### CODE REVIEW

File: clix/flags.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ var Defaults = defaults{
 	ProfilerPort: "8081",
 }
 
-// Flags represents a set of CLI flags.
+// Flags represents a set of Context flags.
```
Comment (rafalmnich) [inline clix/flags.go:55]: Refactoring?

File: clix/context.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ type Context struct {
 	stats  stats.Stats
 }
 
-// NewContext creates a new Context from the CLI Context.
+// NewContext creates a new Context from the Context Context.
```
[the remark this answers is not in the document]
Comment (Skandalik) [reply clix/context.go:40]: Yea...

File: clix/flags.go [hunk 3]
```diff
@@ -3,10 +3,10 @@ package clix
 import (
 	"time"
 
-	"gopkg.in/urfave/cli.v1"
+	"github.com/urfave/cli/v2"
 )
 
-// Flag constants declared for CLI use.
+// Flag constants declared for Context use.
```
[the remark this answers is not in the document]
Comment (Skandalik) [reply clix/flags.go:9]: Yea...

File: clix/flags.go (hunk 1 above)
Comment (Skandalik) [reply clix/flags.go:55]: Yea...

**Pull request merged**